### PR TITLE
Pages and Components added

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Pages and Components added for quiz App . All initial setup done
Screenshot (15)
.

### CODE REVIEW

File: src/Components/Home/QuizType.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                            </div>
+                            <div className="card-footer">
+                                <button className="card-btn-primary">Play Quiz</button>
+                            </div>
+                    </div>
+                </div>
+                <div quiz-type-card>
+                    <div className="card-with-textoverlay card">
+                        <h2 className="card-title xsm-h card-overlay-text sm-h">Football Quiz</h2>
+                        <img className="card-img" src="https://source.unsplash.com/400x400/?football" alt="Image"/>
```
Comment (vaibhav18matere) [inline src/Components/Home/QuizType.jsx:22]: add football-related image here

**Review by vaibhav18matere — APPROVED** [review]
good work, here's small suggestion

**Review by sakleprajwal — APPROVED** [review]
Looks good. please go ahead

File: src/Components/Home/QuizType.jsx (hunk 1 above)
Comment (Mahi24-prog) [reply src/Components/Home/QuizType.jsx:22]: its change randomly

**Pull request merged**